### PR TITLE
Add files used for cloudrun tutorial

### PR DESCRIPTION
**Pull request by ooq**
These files added will be used for a tutorial on how to:
- Run Voucher server as a Cloud Run service
- Invoke the voucher server from a Cloud Build pipeline to sign an built image
We need to merge these files before the tutorial is live on `cloud.google.com`.

### CODE REVIEW

File: tutorials/cloudrun/Dockerfile.bad [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Debian9 image from Jun 8th, 2020
```
Comment (catherinejones) [inline tutorials/cloudrun/Dockerfile.bad]: Would it be possible to move these into a directory called "examples"?

So:

* `tutorials/cloudrun` - everything you need to setup CloudRun
* `tutorials/cloudrun/examples` - examples of how to use the CloudRun setup!

File: tutorials/cloudrun/Dockerfile-server [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Builder
+FROM golang:1.13-alpine as builder
```
Comment (catherinejones) [inline tutorials/cloudrun/Dockerfile-server]: Could we bump this to v1.14 or v1.15?

File: tutorials/cloudrun/Dockerfile-server [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+WORKDIR /go/src/github.com/Shopify/voucher
```
Comment (catherinejones) [inline tutorials/cloudrun/Dockerfile-server]: Can this be updated to `github.com/grafeas/voucher` or do we need to update the module path first?

**Review by catherinejones — CHANGES_REQUESTED** [review]
A few quick changes to request and some questions!

Could we add a README that points (or can be updated to point in the future!) to the CloudRun tutorial?

File: tutorials/cloudrun/Dockerfile.bad (hunk 1 above)
Comment (ooq) [reply tutorials/cloudrun/Dockerfile.bad]: Sounds good! Ack.

File: tutorials/cloudrun/Dockerfile-server (hunk 2 above)
Comment (ooq) [reply tutorials/cloudrun/Dockerfile-server]: Ack.

File: tutorials/cloudrun/Dockerfile-server (hunk 3 above)
Comment (ooq) [reply tutorials/cloudrun/Dockerfile-server]: I shouldn't matter. I will update it.

**Comment by ooq** [comment]
Should we add the README in a separate PR? I think the changes should happen in this order:
- merge this PR
- publish the cloudrun google doc
- add README to point to cloudrun doc
I'll create an issue to track the README.

**Review by catherinejones — APPROVED** [review]
Looks good to me! Waiting to add the README after the tutorial goes live sounds reasonable to me!

**Pull request merged**